examples/Backtesting/Backtesting.md

```diff
@@ -0,0 +1,17 @@
+# Backtesting
+
+These examples demonstrate the backtesting capabilities of BayBE.
+By "backtesting", we refer to the process of setting up a campaign, getting recommendations
+and adding measurements in an iterative fashion.
+
+```{toctree}
+:maxdepth: 1
```

Comment:
This is just a reminder/blocking conversation that we might need to check the levels of the headings in the examples.

Reply:
This was fixed by just making everything in the examples one level higher.